aspnetcore/blazor/call-web-api.md

```diff
@@ -909,7 +909,33 @@ requestMessage.Headers.Add("X-Requested-With", [ "XMLHttpRequest" ]);
 
 Blazor's client-side implementation of <xref:System.Net.Http.HttpClient> uses [Fetch API](https://developer.mozilla.org/docs/Web/API/fetch) and configures the underlying [request-specific Fetch API options](https://developer.mozilla.org/docs/Web/API/fetch#Parameters) via <xref:System.Net.Http.HttpRequestMessage> extension methods and <xref:Microsoft.AspNetCore.Components.WebAssembly.Http.WebAssemblyHttpRequestMessageExtensions>. Set additional options using the generic <xref:Microsoft.AspNetCore.Components.WebAssembly.Http.WebAssemblyHttpRequestMessageExtensions.SetBrowserRequestOption%2A> extension method. Blazor and the underlying Fetch API don't directly add or modify request headers. For more information on how user agents, such as browsers, interact with headers, consult external user agent documentation sets and other web resources.
 
-The HTTP response is typically buffered to enable support for synchronous reads on the response content. To enable support for response streaming, use the <xref:Microsoft.AspNetCore.Components.WebAssembly.Http.WebAssemblyHttpRequestMessageExtensions.SetBrowserResponseStreamingEnabled%2A> extension method on the request.
+:::moniker range=">= aspnetcore-10.0"
+
+Response streaming is enabled by default. To opt-out of response streaming, set an <xref:System.Net.Http.HttpRequestOptionsKey%601> for "`WebAssemblyEnableStreamingResponse`" to `false` on the <xref:System.Net.Http.HttpRequestMessage>:
+
+```csharp
+requestMessage.Options.Set(
+    new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse"), false);
```

Comment:
Since this is the documenation for blazor/aspnet core I would recommend:
```c#
requestMessage.SetBrowserRequestStreamingEnabled(false);
```

Because this extension method exists in the aspnet core components:
https://github.com/dotnet/aspnetcore/blob/83a186cdb2cdd2644b7cb9f44f969a9464f46cc8/src/Components/WebAssembly/WebAssembly/src/Http/WebAssemblyHttpRequestMessageExtensions.cs#L146


I showed the more verbose version in the PR because it was in the runtime repository where this extension method does not exist.
```c#
request.Options.Set(new HttpRequestOptionsKey<bool>("WebAssemblyEnableStreamingResponse"), false);
```

Reply:
I see! I took that toooooo literally!

I'll fix it now. 🏃‍♂️